Library/Formula/ffmpeg.rb

```diff
@@ -33,6 +33,7 @@ class Ffmpeg < Formula
   option "with-libsoxr", "Enable the soxr resample library"
   option "with-webp", "Enable using libwebp to encode WEBP images"
   option "with-zeromq", "Enable using libzeromq to receive commands sent through a libzeromq client"
+  option "with-libbs2b", "Enable using libbs2b"
```

Comment:
I don’t think this line is necessary; Homebrew will generate a similar option text if we don’t explicitly add it.